### PR TITLE
Random swap script for testnet

### PR DESCRIPTION
**Pull request by DevRozaDev**

### CODE REVIEW

File: scripts/package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "setup-testnet": "npx tsc && node --experimental-wasm-modules target/setup-testnet.js",
     "setup-mainnet": "npx tsc && node --experimental-wasm-modules target/setup-mainnet.js",
     "validate-state": "npx tsc && node --experimental-wasm-modules target/validate-state.js",
-    "change-admin": "npx tsc && node --experimental-wasm-modules target/change-admin.js"
+    "change-admin": "npx tsc && node --experimental-wasm-modules target/change-admin.js",
+    "random-swap-testnet": "npx tsc && node --experimental-wasm-modules target/random-swap-testnet.js"
```
Comment (zielvna) [inline scripts/package.json:22]: i guess this script was in `scripts` directory before and that's a leftover?

File: cron/.env [hunk 2]
```diff
@@ -0,0 +1,3 @@
+DEPLOYER_MNEMONIC=
+RECEIVER_ADDRESS=
+NEW_DEPLOYER_MNEMONIC=
```
Comment (zielvna) [inline cron/.env:3]: `RECEIVER_ADDRESS` and `NEW_DEPLOYER_MNEMONIC` are not used in this script

File: cron/src/random-swap-testnet.ts [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+])
+
+const main = async () => {
+  const api = await initPolkadotApi(NETWORK)
+  const keyring = new Keyring({ type: 'sr25519' })
+  const mnemonic = process.env.DEPLOYER_MNEMONIC ?? ''
+  const account = keyring.addFromMnemonic(mnemonic)
+  console.log(`Trader: ${account.address}, Mnemonic: ${mnemonic}`)
+
+  api.tx.utility.batch
```
Comment (zielvna) [inline cron/src/random-swap-testnet.ts:44]: i think this line can be deleted

**Review by zielvna — CHANGES_REQUESTED** [review]

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
Please add small `README.md` file here

File: cron/src/random-swap-testnet.ts [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+      1n << (128n - 1n),
+      true,
+      xToY ? MIN_SQRT_PRICE : MAX_SQRT_PRICE
+    )
+
+    const multiplier = Math.random() * 1.25
+    const amount =
+      (byAmountIn
+        ? simulation.amountIn
+        : simulation.amountOut * BigInt(Math.trunc(multiplier * 100000))) / 100000n
```
Comment (wojciech-cichocki) [inline cron/src/random-swap-testnet.ts:103]: why?

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]

**Review by wojciech-cichocki — APPROVED** [review]

File: cron/README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+```bash
+npm i
+```
+
+### Fill in your mnemonic
+
+In the *.env* file add your own mnemonic in the following line:
+```
+DEPLOYER_MNEMONIC=add your mnemonic here this is the right place for twelve words
```
Comment (zielvna) [inline cron/README.md:27]: it looks like it was change to `USER_MNEMONIC`

**Review by zielvna — APPROVED** [review]

**Pull request merged**